core/server/data/migrations/versions/1.7/1-add-backup-client.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const models = require('../../../../models'),
+    logging = require('../../../../logging'),
+    fixtures = require('../../../schema/fixtures'),
+    _ = require('lodash'),
+    backupClient = fixtures.utils.findModelFixtureEntry('Client', {slug: 'ghost-backup'}),
+    Promise = require('bluebird');
+
+module.exports = function addGhostBackupClient(options) {
+    logging.info('adding backup client');
```

Comment:
For migrations, the message should be a variable.

In the case that the operation is executed, `logging.info(message);` should be called, and vice versa if the operation is not `logging.warn(message);` should be called.

This is a standard, consistent pattern that we use so that it's clear when an operation is done vs skipped.